core/src/main/mima-filters/1.1.x.backwards.excludes/slick.backwards.excludes

```diff
@@ -5,3 +5,38 @@ ProblemFilters.exclude[MissingTypesProblem]("org.apache.pekko.persistence.jdbc.s
 ProblemFilters.exclude[MissingTypesProblem]("org.apache.pekko.persistence.jdbc.snapshot.dao.legacy.SnapshotTables$SnapshotRow$")
 ProblemFilters.exclude[MissingTypesProblem]("org.apache.pekko.persistence.jdbc.state.DurableStateTables$DurableStateRow$")
 ProblemFilters.exclude[FinalMethodProblem]("org.apache.pekko.persistence.jdbc.state.scaladsl.JdbcDurableStateStore.queries")
+ProblemFilters.exclude[IncompatibleSignatureProblem]("org.apache.pekko.persistence.jdbc.db.EagerSlickDatabase.unapply")
```

Comment:
are we sure all these still apply?

would it be possible to put the slick upgrade issues in a separate excludes file?

Reply:
> would it be possible to put the slick upgrade issues in a separate excludes file?

What do you mean by this, its all related? The update from Slick 3.3.x to 3.3.5 is completely binary breaking so all of the excludes are due to the slick upgrade, I don't know how you would split them?

Reply:
@pjfanning In any case I have re-regenerated the `core/src/main/mima-filters/1.1.x.backwards.excludes/slick.backwards.excludes` file to only contain the bare necessary slick excludes that MiMa complains about

Reply:
Oh I see what you mean, `slick.backwards.excludes` [already exists ](https://github.com/apache/incubator-pekko-persistence-jdbc/blob/main/core/src/main/mima-filters/1.1.x.backwards.excludes/slick.backwards.excludes).

This is however an oversight because if you read the currently existing `slick.backwards.excludes` you can see that its referring the Scala 3 update which the previous version of pekko-persistence-jdbc 1.0.x never had so I think it was somehow accidentally committed when it shouldn't have been.

Reply:
Okay I remember the context of the previous PR, we did pekko relevant changes to the API in order for it to compile with Slick 3.5.0 but still releasing with Slick 3.3.0.

So yes it should be a seperate file, I'll do this and add the headers as well